enable/savage/svg/document.py

```diff
@@ -203,17 +203,16 @@ def open_image(self, path):
         renderer.
 
         """
-        path, open = self.resolve(path)
-        fin = open(path)
-
-        from PIL import Image
-        from kiva.compat import piltostring
         import numpy
+        from PIL import Image
+
+        path, open = self.resolve(path)
```

Comment:
`resolve` returns either `lambda x: BytesIO(...)` or `lambda x: open(...)` for the `open` here.